adds starter landing/home page (#5)
papermoonio/polkadot-ecosystem-docs-draft: docs/index.md

---
title: Polkadot Ecosystem Docs
description: Explore the information, tools, and tutorials needed to start building on top of Polkadot, a blockchain network protocol that provides parachains with shared security and interoperability using XCM. 
---

# Polkadot Ecosystem Documentation

Welcome to a draft documentation site for ecosystem-related things on Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. This site aims to serve as a central knowledge exchange for teams across the Polkadot ecosystem. 

> Here, you will find detailed Polkadot information, including key features, tools, and SDKs for parachain and appchain builders. These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.

Author: ```suggestion
Here's what you'll be able to find in the Developer Hub:

- A deep dive into Polkadot - Get familiar with Polkadot's architecture, inner workings, and core technologies, including Agile Coretime, XCM, bridges, governance, and more
- Resources for parachain development - Learn how to create, test, and deploy a parachain from scratch using Substrate and explore how to build and customize your runtime, including adding pallets and implementing custom logic
- Parachain tooling - Access specialized tools and frameworks to streamline parachain development, from launch to maintenance
Documentation on smart contract development - Find resources and tutorials on developing, testing, and deploying smart contracts using Solidity and ink! on Polkadot's EVM and other supported EVM environments in the ecosystem
- Ecosystem tooling - Discover a suite of powerful tools, including SDKs, APIs, and other resources, to build scalable, interoperable applications on Polkadot
- Information on running infrastructure - Infrastructure providers can find detailed information on operating and maintaining validator nodes, including block production, transaction validation, and ensuring network security

These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.
```

> ## What is Polkadot? {: #what-is-polkadot }

Author: ```suggestion

```

> Polkadot is a blockchain network protocol that provides shared security among all parachains and allows all connected tasks to interoperate using XCM. The XCM messaging format will enable blockchains to pass any data to other chains or consensus mechanisms, allowing for secure interoperability and scaling of connected chains. This foundation lets each parachain specialize towards a specific focus and optimize toward that goal without the risk of compromising user security or cross-chain capabilities.

Author: ```suggestion

```

> ## Get Started with Polkadot {: #get-started-with-polkadot }

Author: ```suggestion

```

---
title: Polkadot Ecosystem Docs
description: Explore the information, tools, and tutorials needed to start building on top of Polkadot, a blockchain network protocol that provides parachains with shared security and interoperability using XCM. 
hide:
  - feedback
---

# Polkadot Ecosystem Documentation

> Welcome to the documentation hub for Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. This site aims to serve as a central knowledge exchange for teams across the Polkadot ecosystem. 

Reviewer: ```suggestion
Welcome to the documentation hub for Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. Whether you're creating parachains, developing interoperable tools, or building decentralized applications, this site aims to serve as a central knowledge exchange for developers across the Polkadot ecosystem.
```

> Here, you will find detailed Polkadot information, including key features, tools, and SDKs for parachain and appchain builders. These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.

Reviewer: Oooh thisis good! I'm thinking that we could sort of fill the page out a bit more if we take this second paragraph and create a section that elaborates on the resources and documentation that will be covered on the developer hub. 

What do you think about adding a section like this:

## Comprehensive Developer Resources

Here's what you'll be able to find in the Developer Hub:

- A deep dive into Polkadot - Get familiar with Polkadot's architecture, inner workings, and core technologies, including Agile Coretime, XCM, bridges, governance, and more
- Resources for parachain development - Learn how to create, test, and deploy a parachain from scratch using Substrate and explore how to build and customize your runtime, including adding pallets and implementing custom logic
- Parachain tooling - Access specialized tools and frameworks to streamline parachain development, from launch to maintenance
- Documentation on smart contract development - Find resources and tutorials on developing, testing, and deploying smart contracts using Solidity and ink! on Polkadot's EVM and other supported EVM environments in the ecosystem
- Ecosystem tooling - Discover a suite of powerful tools, including SDKs, APIs, and other resources, to build scalable, interoperable applications on Polkadot
- Information on running infrastructure - Infrastructure providers can find detailed information on operating and maintaining validator nodes, including block production, transaction validation, and ensuring network security

These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.

> ## What is Polkadot? {: #what-is-polkadot }

Reviewer: I think we can get rid of the What is Polkadot and Get Started sections for now

Polkadot is a blockchain network protocol that provides shared security among all parachains and allows all connected tasks to interoperate using XCM. The XCM messaging format will enable blockchains to pass any data to other chains or consensus mechanisms, allowing for secure interoperability and scaling of connected chains. This foundation lets each parachain specialize towards a specific focus and optimize toward that goal without the risk of compromising user security or cross-chain capabilities.

## Get Started with Polkadot {: #get-started-with-polkadot }

### Build a Parachain {: #build-a-parachain }

> - [Paseo Testnet](/paseo-testnet/onboarding) - deploy a custom parachain to the Paseo Testnet

Reviewer: Yes, we did. But I think all of these sections can be removed for right now until we get some more content added

- Resources for parachain development - Learn how to create, test, and deploy a parachain from scratch using Substrate and explore how to build and customize your runtime, including adding pallets and implementing custom logic
- Parachain tooling - Access specialized tools and frameworks to streamline parachain development, from launch to maintenance
Documentation on smart contract development - Find resources and tutorials on developing, testing, and deploying smart contracts using Solidity and ink! on Polkadot's EVM and other supported EVM environments in the ecosystem
- Ecosystem tooling - Discover a suite of powerful tools, including SDKs, APIs, and other resources, to build scalable, interoperable applications on Polkadot
- Information on running infrastructure - Infrastructure providers can find detailed information on operating and maintaining validator nodes, including block production, transaction validation, and ensuring network security

These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.

> ### Build a Parachain {: #build-a-parachain }

Author: ```suggestion

```

> - [Paseo Testnet](/paseo-testnet/onboarding) - deploy a custom parachain to the Paseo Testnet

Author: ```suggestion

```

> ### Develop Smart Contracts {: develop-smart-contracts}

Author: ```suggestion

```

> ### Create Tooling 

Author: ```suggestion

```

> ### Validate and Produce Blocks

Author: ```suggestion
```

> <!--TODO: Rename this? Need a name for this Infrastructure section-->

Author: ```suggestion

```
